Content.Server/_NF/Medical/SuitSensors/DisableSuitSensorComponent.cs

```diff
@@ -0,0 +1,19 @@
+
+namespace Content.Server._NF.Medical.SuitSensors;
+
+// A component to disable suit sensors, regardless of settings, for a particular entity (e.g. medical corpses)
+[RegisterComponent]
+public sealed partial class DisableSuitSensorComponent : Component
```

Comment:
```suggestion
public sealed partial class DisableSuitSensorsComponent : Component
```
Keep the component name the same, this is a breaking change we don't need.

Also rename the file :pray: